FP-Erlang/Week 6/Assignment 5/Mishal/assignment5.erl

```diff
@@ -0,0 +1,37 @@
+-module (assignment5).
+-export ([add/1,subtract/1,multiply/1,divide/1,greaterThanX/1,lessThanX/1,three_a/2,three_b/1,three_c/1,derive/2,integral/4,foldl/3]).
+
+%%%%%%%%%%%%%%%%%%%%%%%%  QUESTION 1  %%%%%%%%%%%%%%%%%%%%%%%%
+
+% Part 1 %
+add(X) -> fun(Y) -> X+Y end.
+subtract(X) -> fun(Y) -> X-Y end.
+multiply(X) -> fun(Y) -> X*Y end.
+divide(X) -> fun(Y) -> X/Y end.
```

Comment:
Include the two argument versions of add, subtract, multiply and divide by making calls to the curried functions